.travis.yml

```diff
@@ -4,4 +4,4 @@ services:
   - docker
 
 script:
-  - docker run -v $(pwd):/pkg hobarrera/makepkg
+  - docker run -v $(pwd):/pkg neomutt/aur
```

Comment:
Uhm... the source for this is hidden, but it _looks_ like a copy-paste of `hobarrera/makepkg`. Would you care to explain that?

Reply:
The docker image was stale when I tried to build this using hobarrera/makepkg. It refused to recognise a valid dependency. @flatcap said in IRC to fork hobarrera/makepkg into https://github.com/neomutt/aur-docker so rebuilds can be triggered if it goes stale again.

Reply:
I'm not sure how this was set up, but Docker isn't showing that repository as the source for the `Dockerfile`. In fact, it's not showing any source, or anything. Maybe you used another, private repository, or some other mechanism to build the Docker images?

In any case, I can set up my `makepkg` repo to trigger builds daily. Since it's used for other packages' CI, I'd prefer to avoid having a duplicate (if possible).